hoot-core-test/src/test/cpp/hoot/core/io/ServiceOsmApiDbBulkInserterTest.cpp

```diff
@@ -117,7 +117,7 @@ class ServiceOsmApiDbBulkInserterTest : public CppUnit::TestFixture
       (long)1,
       DbUtils::getRowCount(reader._getDatabase()->getDB(), ApiDb::getRelationsTableName()));
     CPPUNIT_ASSERT_EQUAL(
-      (long)2,
+      (long)3,
```

Comment:
Database tables contain the extra `type` tag for relations, hence all of the changes from `2` to `3`.